packages/nextjs/app/builds/_components/AllBuilds.tsx

```diff
@@ -143,8 +180,8 @@ export function AllBuilds({ searchParams }: { searchParams: { category?: BuildCa
               </>
             )}
             {builds &&
```

Comment:
While scrolling down and fetching next set of builds maybe we show user the message  like loading or fetching next builds?